cli/args/config_file.rs

```diff
@@ -350,6 +351,17 @@ impl FilesConfig {
   }
 }
 
+impl Add for FilesConfig {
```

Comment:
Let's make this an explicit method instead of implementing `Add` so it's more discoverable and so we can mutate to reduce the amount of cloning we do.